From 4aa3a62b38196f846a2d10b94b03aef6ca82c6b0 Mon Sep 17 00:00:00 2001
From: Stephan Hartmann <stha09@googlemail.com>
Date: Mon, 8 Nov 2021 21:02:15 +0000
Subject: [PATCH] GCC: make gfx::Point::operator== constexpr

blink::IntRect::operator== is constexpr. Therefore,
gfx::Point::operator== needs to be constexpr as well.

Bug: 819294
Change-Id: I148c79c19cac971c4d67b868b24e1b606e14e9ca
Reviewed-on: https://chromium-review.googlesource.com/c/chromium/src/+/3263131
Reviewed-by: Scott Violet <sky@chromium.org>
Commit-Queue: Stephan Hartmann <stha09@googlemail.com>
Cr-Commit-Position: refs/heads/main@{#939512}
---
 ui/gfx/geometry/point.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/ui/gfx/geometry/point.h b/ui/gfx/geometry/point.h
index 71c5a5b949bb1..55b3220895407 100644
--- a/ui/gfx/geometry/point.h
+++ b/ui/gfx/geometry/point.h
@@ -95,7 +95,7 @@ class GEOMETRY_EXPORT Point {
   int y_;
 };
 
-inline bool operator==(const Point& lhs, const Point& rhs) {
+constexpr bool operator==(const Point& lhs, const Point& rhs) {
   return lhs.x() == rhs.x() && lhs.y() == rhs.y();
 }
 
-- 
2.34.1

